README.md

```diff
@@ -49,16 +49,30 @@ Linux build is disabled currently because of unmet dependecies with some distros
 ### Development
 
 1. Install node.js
-2. Change to project folder and run `npm install`
-3. Run `npm start`
+2. Change to project folder and run `npm install`.
+3. Run `npm start`.
+
+4. Optional, install gulp `npm install --global gulp-cli`.
```

Comment:
This should rather go into the `App build and release` section, as it's only needed to build the app.